src/components/DelegationActionBar/DelegationActionBar.tsx

```diff
@@ -0,0 +1,115 @@
+import { Button, List, ListItem } from '@digdir/design-system-react';
+import { useState } from 'react';
+import { t } from 'i18next';
+import * as React from 'react';
+
+import { ReactComponent as MinusCircle } from '@/assets/MinusCircle.svg';
+import { ReactComponent as AddCircle } from '@/assets/AddCircle.svg';
+
+import { Line } from '../Line';
+import { ActionBar, type ActionBarProps } from '../ActionBar';
+
+import classes from './DelegationActionBar.module.css';
+
+export interface DelegationActionBarProps extends Pick<ActionBarProps, 'color'> {
+  title?: string;
+  subtitle?: string;
+  topContentText?: string;
+  bottomContentText?: string;
+  textList?: string[];
+  buttonType: 'add' | 'remove';
+  onActionButtonClick: () => void;
+}
+
+export const DelegationActionBar = ({
+  title = 'No info',
+  subtitle = 'No info',
+  topContentText,
+  bottomContentText,
+  textList = [''],
+  buttonType,
+  onActionButtonClick,
+  color = 'neutral',
+}: DelegationActionBarProps) => {
+  const [open, setOpen] = useState(false);
+
+  const actions = (
+    <>
+      {buttonType === 'add' && (
+        <Button
+          icon={<AddCircle />}
+          variant={'quiet'}
+          color={'success'}
+          onClick={onActionButtonClick}
+          aria-label={'soft-add'}
+        ></Button>
+      )}
+      {buttonType === 'remove' && (
+        <Button
+          icon={<MinusCircle />}
+          variant={'quiet'}
+          color={'danger'}
+          onClick={onActionButtonClick}
+          aria-label={'soft-remove'}
+        ></Button>
+      )}
+    </>
+  );
+
+  return (
+    <div className={classes.delegationActionBar}>
+      <ActionBar
+        title={<p className={classes.actionBarHeaderTitle}>{title}</p>}
+        subtitle={subtitle}
+        actions={actions}
+        color={color}
+        open={open}
+        onClick={() => {
+          setOpen(!open);
+        }}
+      >
+        {' '}
+        <div className={classes.newApiAccordionContent}>
+          {textList.length > 0 && (
+            <div>
+              <p className={classes.scopeText}>{t('api_delegation.scopes')}:</p>
+              <List borderStyle='dashed'>
+                {textList.map((scope, i) => {
```

Comment:
I'm not in love with the name textList when it pertains to scopes in a component that is this specialized anyway, but if we wish to keep the name as generalized as textList, we should probably stick to that when referring to the list items in the map function.

That being said, I actually think it is more readable when referring to these as scopes, so my favored solution would be to change textList into scopeList. The most important though, is that we are consistant.

Reply:
fixed